packages/theme/getStore.ts

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import urlJoin from 'url-join';
+
+interface StoreConfig {
+  backendUrl: string
+}
+
+/* eslint-disable camelcase */
+export interface StoreAttr {
+  type: string
+  id: string
+  attributes: {
+    name: string
+    url: string
+    meta_description: string | null
+    meta_keywords: string | null
+    seo_title: string | null
+    default_currency: string
+    supported_currencies: string
+    default_locale: string
+    supported_locales: string
+  }
+}
+interface IStore {
```

Comment:
My first thought - why are there two Store interfaces? Consider distinguishing between the deserialized Store and store's response. 

`Store` as en entity is fine . I'd only replace `IStore` with `StoreResponse`.

Reply:
Right, I tried to recreate the naming structure present in the SDK but it doesn't work here. Will correct.